cmd/bridge/main.go

```diff
@@ -235,6 +236,7 @@ func main() {
 		DevCatalogCategories:  *fDevCatalogCategories,
 		UserSettingsLocation:  *fUserSettingsLocation,
 		EnabledConsolePlugins: consolePluginsMap,
+		AddPageJSON:           *fAddPageJSON,
```

Comment:
Let's drop the JSON suffix here as well.

Reply:
Ack. Removed `JSON` and `-json` suffixes